engine/message/db.go

```diff
@@ -590,13 +600,40 @@ func (db *DB) _SendMessages(ctx context.Context, send SendFunc, status StatusFun
 	}
 
 	var msgs []msgMeta
-	for rows.Next() {
-		var msg msgMeta
-		err = rows.Scan(&msg.MessageID, &msg.AlertID, &msg.UserID, &msg.CMID)
+
+	// if twilio is disable, create an entry to notify the user
+	cfg := config.FromContext(ctx)
+	if !cfg.Twilio.Enable {
+		rows, err := tx.StmtContext(ctx, db.failSMSVoice).QueryContext(execCtx)
+		if err != nil {
+			return errors.Wrap(err, "check for failed message")
+		}
+		defer rows.Close()
+
+		for rows.Next() {
+			var msg msgMeta
+			err = rows.Scan(&msg.MessageID, &msg.AlertID, &msg.UserID, &msg.CMID)
+			if err != nil {
+				return errors.Wrap(err, "scan all failed messages")
+			}
+			msgs = append(msgs, msg)
+		}
+	} else {
```

Comment:
The `failDisabledCM` code must always be run, regardless of the state of Twilio -- should not be inside an `else`